screenpipe-app-tauri/components/search-chat.tsx

```diff
@@ -359,7 +366,10 @@ export function SearchChat() {
       role: "user" as const,
       content: floatingInput,
     };
-    setChatMessages((prevMessages) => [...prevMessages, userMessage]);
```

Comment:
<img width="1312" alt="Screenshot 2024-11-01 at 5 36 59 PM" src="https://github.com/user-attachments/assets/63f2a08a-b88f-49c9-9b92-bc84e30c79ec">

not sure this is correct

this creates an empty chat in the UI

Reply:
![Uploading Screenshot 2024-11-01 at 5.37.48 PM.png…]()

Reply:
<img width="1312" alt="Screenshot 2024-11-01 at 5 37 48 PM" src="https://github.com/user-attachments/assets/14c2ac7b-9a81-410c-9dae-e2d10e883775">